client/src/ui/atoms/logo/index.tsx

```diff
@@ -13,12 +14,20 @@ export function Logo() {
   const locale = useLocale();
   const location = useLocation();
 
-  const plus = isPlus(location.pathname);
-  const docs = isDocs(location.pathname);
+  const [mdnLogo, setMdnLogo] = React.useState<React.ReactNode>(<MDNLogo />);
```

Comment:
hey @OnkarRuikar 👋, looks amazing. i tested it out but unfortunately, making `<MDNLogo />` as a default isn't solving the hydration error in logo. however, making it `null` is works but with a caveat; it is creates a layout shifting problem.

Reply:
Hm, setting it to `null` also means users with disabled JavaScript won't see a logo.

Could we just set `suppressHydrationWarning` on the Logo wrapper?

Reply:
@caugner I also tried it but unfortunately, no. `suppressHydrationWarning` only works one level deep. as far as I understand, hydration error is happening inside `<svg>`.

in this case, we have to set `suppressHydrationWarning` on the `<svg>` but we can't since it is not a react component.